README.rst

```diff
@@ -110,3 +110,34 @@ Installing the Images
 =====================
 
 See detailed instructions in doc/howtos/image-install.rst.
+
+Mailing Lists
+=============
+
+The project has a mailing list for all project development related discussions.
+
+The mailing list info/subscription page can be found `here`_
+
+.. _here: https://lists.yoctoproject.org/listinfo/intel-iot-refkit
+
+Bug Tracking
+============
+
+The project uses the `Yocto Project Bugzilla`_ for feature/enhancement and bug tracking.
+
+If you've found a bug or wish to file an enhancement request, go to Bugzilla
+and submit the tickets against Yocto Project Subprojects | IoT Reference OS Kit
+component.
```

Comment:
We could also use this URL to make it easier for users to file a bug or feature request for the IoT Reference OS Kit: https://bugzilla.yoctoproject.org/enter_bug.cgi?product=IoT%20Reference%20OS%20Kit